packages/chakra-components/src/MenuBar/SubMenu.js

```diff
@@ -159,26 +136,13 @@ const SubMenu = ({
     resetTabIndex();
   };
 
-  const closeMenuWithoutIndex = () => {
-    if (!isControlled) {
-      setIsOpen(false);
-    }
-
-    if (onClose) {
-      onClose();
-    }
-
-    resetTabIndex();
-  };
-
```

Comment:
Why was this removed?

Reply:
Seems this is not yer added back.
Please add it bro.

Reply:
added